packages/modules/src/modules/banners/designableBanner/DesignableBanner.tsx

```diff
@@ -42,54 +42,56 @@ const DesignableBanner: ReactComponent<BannerRenderProps> = ({
         return <></>;
     }
 
+    const { basic, primaryCta, secondaryCta, highlightedText, closeButton } = design.colours;
+
     const templateSettings: BannerTemplateSettings = {
         containerSettings: {
-            backgroundColour: hexColourToString(design.colours.basic.background),
-            textColor: hexColourToString(design.colours.basic.bodyText),
+            backgroundColour: hexColourToString(basic.background),
+            textColor: hexColourToString(basic.bodyText),
         },
         headerSettings: {
-            textColour: neutral[100],
+            textColour: hexColourToString(basic.headerText),
         },
         primaryCtaSettings: {
             default: {
-                backgroundColour: brand[500],
-                textColour: neutral[100],
+                backgroundColour: hexColourToString(primaryCta.default.background),
+                textColour: hexColourToString(primaryCta.default.text),
             },
             hover: {
-                backgroundColour: neutral[100],
-                textColour: brand[500],
+                backgroundColour: hexColourToString(primaryCta.hover.background),
+                textColour: hexColourToString(primaryCta.hover.text),
             },
         },
         secondaryCtaSettings: {
             default: {
-                backgroundColour: specialReport[100],
-                textColour: neutral[100],
-                border: `1px solid ${neutral[100]}`,
+                backgroundColour: hexColourToString(secondaryCta.default.background),
+                textColour: hexColourToString(secondaryCta.default.text),
+                border: `1px solid ${secondaryCta.default.border || neutral[100]}`,
```

Comment:
Does this need a `hexColourToString`? (I'm assuming `secondaryCta.default.border` is an HexColour?)

Reply:
good spot!